utils/common/status_dns.yml

```diff
@@ -0,0 +1,41 @@
+- name: Pre chaos checks
+  block:
+    - name: Creating an nginx pod
+      shell: kubectl run nginx --image=nginx --generator=run-pod/v1 -n {{ namespace }}
+
+    - name: Checking whether application pods are in running state
+      shell: kubectl get pod nginx -n default -o custom-columns=:.status.phase --no-headers -n {{ namespace }}
+      register: result
+      until: "((result.stdout.split()|unique)|length) == 1 and 'Running' in result.stdout"
+      retries: 10
+
+    - name: Exposing the nginx pod to port 80
+      shell: kubectl expose pod nginx --port 80 -n {{ namespace }}
+
+    - name: Creating a curl-pod to curl the nginx servicce
+      shell: kubectl run curl-pod --image radial/busyboxplus:curl --generator run-pod/v1 -- sleep 1000
+  when: check == "pre"
+
```

Comment:
Why not use a jinja template with the pod spec that we just apply - will reduce the ansible tasks